amqphandler/amqphandler.go

```diff
@@ -741,3 +762,19 @@ func (handler *AmqpHandler) createCloudMessage(messageType string, data interfac
 		Data: data,
 	}
 }
+
+func (handler *AmqpHandler) notifyCloudConnected() {
+	handler.isConnected = true
```

Comment:
method notify shouldn't change the internal state of the object

Reply:
fixed